pkg/sparrow/run.go

```diff
@@ -62,6 +74,13 @@ func New(cfg *config.Config) *Sparrow {
 		cCfgChecks:  make(chan map[string]any, 1),
 		routingTree: api.NewRoutingTree(),
 		router:      chi.NewRouter(),
+		targets: targets.NewGitlabManager(
+			gitlab.New(gitlabBaseUrl, os.Getenv("GITLAB_TOKEN"), gitlabRegistrationProjectID),
+			"cool-sparrow.de",
+			globalTargetsCheckInterval,
+			registrationUnhealthyThreshold,
+			registrationInterval,
+		),
```

Comment:
We need to be able to configure the settings with the startup-config

Reply:
Done